xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -284,15 +275,9 @@ void CGUIDialogPVRTimerSettings::InitializeSettings()
   if (tag->SupportsFolders())
     AddEdit(group, SETTING_TMR_DIR, 19076, 0, tag->m_strDirectory, true, false, 19104);
 
-  AddToggle(group, SETTING_TMR_RADIO, 19077, 0, tag->m_bIsRadio);
```

Comment:
it's common practice for skins to use strings from our language file.
confluence does it and so does every other skin out there.

if we ever want to remove a string, just gimme a ping and i'll check the skins in our repo whether they would be affected or not.

i've checked 19077 and no problem there.